crates/oxc_wasm/src/lib.rs

```diff
@@ -204,6 +212,13 @@ impl Oxc {
             self.formatted_text = printed;
         }
 
+        if run_options.prettier_format() {
+            let printed =
+                Prettier::new(&allocator, source_text, trivias, PrettierOptions::default())
```

Comment:
We can probably expand later and add more options to configure Prettier behaviour

Reply:
The playground needs so many knobs!